client/src/contexts/index.tsx

```diff
@@ -2,13 +2,25 @@ import React, { ReactElement } from "react";
 import { SocketContext, SocketProvider } from "./SocketIOContext";
 import { DeviceContext, DeviceProvider } from "./DeviceContext";
 import { UserInfoContext, UserInfoProvider } from "./UserInfoContext";
+import { PositionsContext, PositionsProvider } from "./PositionsContext";
+import { ThemeProvider } from "atomize";
+
+const theme = {
+  fontFamily: {
+    primary: "DM Sans, sans-serif",
+  },
+};
 
 const Providers = ({ children }: { children: ReactElement }): JSX.Element => (
-  <SocketProvider>
-    <DeviceProvider>
-      <UserInfoProvider>{children}</UserInfoProvider>
-    </DeviceProvider>
-  </SocketProvider>
+  <ThemeProvider theme={theme}>
+    <SocketProvider>
+      <PositionsProvider>
```

Comment:
It was removed here
https://github.com/Amijakan/mochatto/pull/126/commits/74fc56490396c51dc76f18dd04a5d9569929308e